src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -315,7 +335,42 @@ class EduSharingConnector {
 
 			return new EduSharingResponse(parsed);
 		} catch (err) {
-			logger.error('Edu-Sharing failed search ', err.message);
+			logger.error(`Edu-Sharing failed search: ${err.message}`);
```

Comment:
Same point like i mentioned in the button of the file.

Reply:
Change it in new commit.